test/selenium/README.md

```diff
@@ -22,6 +22,62 @@ Run the [run-tests script](https://github.com/cockpit-project/cockpit/blob/maste
 
 Although this is the default way to run selenium tests the run-tests script is configurable and can be changed to run tests against different machines. This can be useful for developing or debugging tests. Check bellow the HACKING section for more details.
 
+### Debugging tests:
+Selenium testsuites via ``run-tests`` allows possibility how to debug tests.
+  - Let machines running via ``--sit`` option what let all machines running after tests fisnish
```

Comment:
- Pass ``-sit`` parameter to ``run-tests`` which will leave all test machines running after the tests finish.